packages/@expo/cli/src/lint/index.ts

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+import chalk from 'chalk';
+
+import { Command } from '../../bin/cli';
+import { assertArgs, getProjectRoot, printHelp } from '../utils/args';
+
+export const expoLint: Command = async (argv) => {
+  const args = assertArgs(
+    {
+      // Types
+      '--help': Boolean,
+      // Aliases
+      '-h': '--help',
+    },
+    argv
+  );
+
+  if (args['--help']) {
+    printHelp(
+      `Utility for setting up linting on an Expo project`,
+      chalk`npx expo lint`,
+      [`-h, --help                 Usage info`].join('\n')
+    );
+  }
+
+  // Load modules after the help prompt so `npx expo lint -h` shows as fast as possible.
+  const { lintAsync } = require('./lintAsync') as typeof import('./lintAsync');
```

Comment:
Prefer modern ESM syntax for these types of lazy loading:

```suggestion
  const { lintAsync } = await import('./lintAsync.js');
```